packages/create-plugin/src/commands/update.command.ts

```diff
@@ -8,54 +8,79 @@ import {
   updateNpmScripts,
 } from '../utils/utils.npm';
 
+async function updateConfigurationFiles() {
+  if (await confirmPrompt(TEXT.updateConfigPrompt)) {
+    compileTemplateFiles(UDPATE_CONFIG.filesToOverride, getTemplateData());
+    printSuccessMessage(TEXT.overrideFilesSuccess);
+  } else {
+    printMessage(TEXT.overrideFilesAborted);
+    process.exit(0);
+  }
+}
+
+async function updateNpmDependencies() {
+  if (hasNpmDependenciesToUpdate()) {
+    //  ask if we should update the grafana dependencies
+    const shouldUpdateGrafanaPackages = await confirmPrompt(TEXT.updateNpmGrafanaPackagesPrompt);
+
+    const updatadableDependenciesText = getPackageJsonUpdatesAsText({
```

Comment:
This part fixes the issue on updating grafana dependencies by default. Now it will prompt the user to confirm if they want to do so.